core/src/main/java/tech/pegasys/eth2signer/core/http/SigningRequestHandler.java

```diff
@@ -43,11 +43,12 @@ public SigningRequestHandler(
   @Override
   public void handle(final RoutingContext context) {
     LOG.debug("Received a request for {}", context.normalisedPath());
-    generateResponseFromBody(context.response(), context.getBody());
+    final String publicKey = context.pathParam("param1");
```

Comment:
where does the "param1" comes from? Is it specific with VertX?

Reply:
yes, that's how vertx names the params

Reply:
👍

Reply:
I think the regex needs to live in this class as a static variable.